src/Azure.Functions.Cli/Actions/LocalActions/InstallExtensionAction.cs

```diff
@@ -75,9 +81,14 @@ public async override Task RunAsync()
 
                 if (string.IsNullOrEmpty(Package) && string.IsNullOrEmpty(Version))
                 {
+                    var project = ProjectHelpers.GetProject(extensionsProj);
                     foreach (var extensionPackage in ExtensionsHelper.GetExtensionPackages())
                     {
-                        await AddPackage(extensionsProj, extensionPackage.Name, extensionPackage.Version);
+                        // Only add / update package referece if it does not exist or forced update is enabled
```

Comment:
I don't quite understand this comment. Why only add/update if it doesn't exist? wouldn't `dotnet` just takes care of doing what ever is right?

Reply:
When you run 'dotnet add' it always updates the extension version in the csproj, even if the package reference is already present.
This change would only add extension package if it is already not present.